src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1332,6 +1333,16 @@ void uct_dc_mlx5_ep_handle_failure(uct_dc_mlx5_ep_t *ep, void *arg,
     }
 }
 
+static void
+uct_dc_mlx5_ep_send_handler(uct_rc_iface_send_op_t *op, const void *resp)
```

Comment:
uct_dc_mlx5_ep_check_send_completion

Reply:
renamed to uct_dc_mlx5_ep_check_send_completion